custom/conf/app.example.ini

```
@@ -1923,6 +1923,19 @@ PATH =
 ;SCHEDULE = @every 168h
 ;OLDER_THAN = 8760h
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Delete all old actions from database
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[cron.delete_old_actions]
```

Shouldn't this be something related to updates? I guess it's a copy/paste thingy. Same for the comment above and the cheetsheet changes.

---

```suggestion
 ;[cron.update_checker]
```
haha yup, bitten by copy/paste